src/restaurant_inspections_api/handlers/inspections.clj

```diff
@@ -1,53 +1,45 @@
 (ns restaurant-inspections-api.handlers.inspections
   (:require [restaurant-inspections-api.validations :as validate]
             [restaurant-inspections-api.util :as util]
-            [restaurant-inspections-api.services :as srv]))
+            [restaurant-inspections-api.services :as srv]
+            [restaurant-inspections-api.db :as db]
+            [restaurant-inspections-api.constants :as constants]
+            [clojure.string :as str]))
 
 (defn validate-inspections-params
   "Receives all inspections query parameters (nil when not specified) and returns a map of valid and
   invalid params."
-  [zip-codes business-name start-date end-date district-code county-number per-page page]
+  [{:keys [zipCodes businessName startDate endDate districtCode countyNumber perPage page]}]
```

Comment:
meh.. yeah no camel case. we dont want to be mixing camel and dashes on params/variables